xml/admin_monitoring_alerting.xml

```diff
@@ -1385,79 +1381,79 @@ monitoring:
             that has been acquired by BlueFS and is considered free. Value
             <option>available_from_bluestore</option> denotes ability of &bluestore; to leave
             more space to BlueFS. It is normal that this value is different from
             amount of &bluestore; free space, as BlueFS allocation unit is
             typically larger than &bluestore; allocation unit. This means that
-            only part of &bluestore; free space will be acceptable for BlueFS.</para>
+            only part of &bluestore; free space is acceptable for BlueFS.</para>
         </listitem>
       </varlistentry>
       <varlistentry>
         <term><option>BLUEFS_LOW_SPACE</option></term>
         <listitem>
           <para>If BlueFS is running low on available free space and there is little
-            <option>available_from_bluestore</option> you can consider reducing BlueFS allocation
-            unit size. To simulate available space when allocation unit is different do:</para>
+            <option>available_from_bluestore</option>, consider reducing BlueFS allocation
+            unit size. To simulate available space when allocation unit is different, execute:</para>
 <screen>&prompt.cephuser;ceph daemon osd.123 bluestore bluefs available <replaceable>ALLOC-UNIT-SIZE</replaceable></screen>
         </listitem>
       </varlistentry>
       <varlistentry>
         <term><option>BLUESTORE_FRAGMENTATION</option></term>
         <listitem>
-          <para>As &bluestore; works, free space on underlying storage will get
-            fragmented. This is normal and unavoidable but excessive fragmentation
-            will cause slowdown. To inspect &bluestore; fragmentation one can do:</para>
+          <para>As &bluestore; works, free space on underlying storage becomes
+            fragmented. This is normal and unavoidable, but excessive fragmentation
+            can cause slowdown. To inspect &bluestore; fragmentation, execute:</para>
 <screen>&prompt.cephuser;ceph daemon osd.123 bluestore allocator score block</screen>
           <para>Score is given in [0-1] range. [0.0 .. 0.4] tiny fragmentation [0.4 .. 0.7]
             small, acceptable fragmentation [0.7 .. 0.9] considerable, but safe
-            fragmentation [0.9 .. 1.0] severe fragmentation, may impact BlueFS
+            fragmentation [0.9 .. 1.0] severe fragmentation, can impact BlueFS
             ability to get space from &bluestore;.
-            If detailed report of free fragments is required do:</para>
+            If detailed report of free fragments is required, execute:</para>
 <screen>&prompt.cephuser;ceph daemon osd.123 bluestore allocator dump block</screen>
-          <para>In case when handling OSD process that is not running fragmentation
-           can be inspected with <command>ceph-bluestore-tool</command>. Get fragmentation score:</para>
+          <para>If the OSD process does not perform fragmentation, it can be inspected with
```

Comment:
I think we try to avoid passive voice.

"If the X does not do Y, then you can inspect it with"